playbooks/reboot_cluster.yml

```diff
@@ -11,6 +11,8 @@
   - name: waiting for server to come back
     wait_for: host={{ inventory_hostname }}
               state=started
+              connect_timeout=900
```

Comment:
Connect_timeout is the maximum amount of time to wait for the ssh connection, while timeout is the maximum amount of time to wait for the task to complete.  Looking at this again, I don't think connect_timeout actually needs to be this high.